Xamarin.Forms.Core/GestureRecognizer.cs

```diff
@@ -1,9 +1,209 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
 namespace Xamarin.Forms
 {
 	public class GestureRecognizer : Element, IGestureRecognizer
 	{
 		internal GestureRecognizer()
 		{
 		}
+
+		public static readonly BindableProperty StateProperty =
```

Comment:
if we use `TouchCount` why not use `TouchState`? or drop `Touch` from both ?

Reply:
done